core/app/[locale]/(default)/product/[slug]/_components/wishlist-button/add-to-new-wishlist-modal.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
```

Comment:
I wonder if `wishlist-button` (all components) should live under the `@/vibes/soul/section/product-detail` section.

Reply:
@jorgemoya I had a brief discussion about this in DMs with Miguel. This particular wishlist implementation is very specific to BC, so we opted for a "slotted" pattern where we just pass in the additional component through the `additionalActions` prop on the ProductDetails.

Reply:
I understand but this kind of goes against the themeing work we want to implement. We need to stop thinking about `vibes/soul` and more like `ui`. cc @migueloller